build-logic/performance-testing/src/main/groovy/gradlebuild/performance/reporter/PerformanceReporter.groovy

```diff
@@ -66,7 +66,7 @@ class PerformanceReporter {
                 spec.args(reportDir.path, projectName)
                 spec.args(resultJsons*.path)
                 spec.systemProperties(databaseParameters)
-                spec.debug = debugReportGeneration
+                spec.debugOptions.enabled.set(debugReportGeneration)
```

Comment:
❔ Can we use `=` instead of `set()` here?

Reply:
As I remember we have to use `.set` here, since we use `@CompileStatic`. I will double check

Reply:
Yes, since we use `@CompileStatic` we can't use `=` here